Content.Client/Paper/UI/PaperWindow.xaml.cs

```diff
@@ -252,6 +253,30 @@ public void Populate(PaperComponent.PaperBoundUserInterfaceState state)
             var msg = new FormattedMessage();
             msg.AddMarkupPermissive(state.Text);
 
+            // DeadSpace
```

Comment:
```suggestion
            // DS14-signatures-start
```
И в конце // DS14-signatures-end